tests/nativeapp/test_manager.py

```diff
@@ -199,7 +199,10 @@ def test_drop_object_no_show_object(mock_execute, temp_dir, mock_cursor):
             (mock_cursor(["row"], []), mock.call("use role sample_package_role")),
             (
                 mock_cursor([], []),
-                mock.call("show application packages like 'SAMPLE_PACKAGE_NAME'"),
+                mock.call(
+                    "show application packages like 'SAMPLE_PACKAGE_NAME'",
+                    cursor_class=DictCursor,
```

Comment:
The assert call below actually never got reached, which is why this had to change (even though the underlying code did not).